docs/standard/assembly/index.md

```diff
@@ -1,61 +1,115 @@
 
-In .NET Core and .NET Framework, an assembly can be built from one or more source code files. In .NET Framework, assemblies can contain one or more modules. This allows larger projects to be planned in such a way that several individual developers work on separate source code files or modules, which are combined to create a single assembly. For more information about modules, see [How to: Build a Multifile Assembly](../../framework/app-domains/how-to-build-a-multifile-assembly.md).
+In .NET Core and .NET Framework, you can build an assembly from one or more source code files. In .NET Framework, assemblies can contain one or more modules. This allows larger projects to be planned so that several developers can work on separate source code files or modules, which are combined to create a single assembly. For more information about modules, see [How to: Build a multifile assembly](build-multifile.md).
 
 Assemblies have the following properties:
 
-- Assemblies are implemented as .exe or .dll files.
+- Assemblies are implemented as *.exe* or *.dll* files.
 
-- For libraries that target the .NET Framework, you can share an assembly between applications by putting it in the [global assembly cache](../../framework/app-domains/gac.md). Assemblies must be strong-named before they can be included in the global assembly cache. For more information, see [Strong-Named Assemblies](../../framework/app-domains/strong-named-assemblies.md).
+- For libraries that target the .NET Framework, you can share assemblies between applications by putting them in the [global assembly cache (GAC)](../../framework/app-domains/gac.md). You must strong-name assemblies before you can include them in the GAC. For more information, see [Strong-named assemblies](strong-named.md).
 
-- Assemblies are only loaded into memory if they are required. If they are not used, they are not loaded. This means that assemblies can be an efficient way to manage resources in larger projects.
+- Assemblies are only loaded into memory if they are required. If they aren't used, they aren't loaded. This means that assemblies can be an efficient way to manage resources in larger projects.
 
 - You can programmatically obtain information about an assembly by using reflection. For more information, see [Reflection (C#)](../../csharp/programming-guide/concepts/reflection.md) or [Reflection (Visual Basic)](../../visual-basic/programming-guide/concepts/reflection.md).
 
-- You can load an assembly only to inspect by using the <xref:System.Reflection.MetadataLoadContext> class.
-
-## Assembly manifest
+- You can load an assembly just to inspect it by using the <xref:System.Reflection.MetadataLoadContext> class.
```

Comment:
@rpetrusha Thanks for your review. It will probably take quite a few days to make these changes. I'm trying to think of a sensible way to do this so that this PR can get merged asap. It seems like quite a few of the requested content changes are to preexisting content. If we can just merge this PR once the move and combining changes are final, we could then make the content changes in smaller, more sane PRs. That would make things much easier.  
For now, I'll start out with re-moving or un-moving and un-renaming content that you don't want to move after all or that you want to move elsewhere. 
Once the move and renaming are final, maybe we could merge this PR, and I can then proceed with the content and code changes you want. Thanks.

Reply:
@rpetrusha - needed %2A.